cmd/autothrottle/api_test.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"fmt"
+	"github.com/DataDog/kafka-kit/v3/kafkazk"
```

Comment:
Can you move this to a line-separated entry after the std libs? Going to add a linter to CI and it will complain